lib/src/utility/apps/files/links.py

```diff
@@ -36,6 +37,7 @@ def get_links_to_files(df: DF, ctx: Context):
     _expire_secs = ctx.app_cfg.get('expiration', 6 * 3600)
     _expire_secs = min(_expire_secs, 24 * 3600)
     _expire_secs = max(_expire_secs, 0)
+    is_asset = True if 'path' in df.columns else False
```

Comment:
There can be a dataframe that has this column. As we are not imposing DF[obj], the processor should work with an arbitrary tables as well. By making this condition, you will end up failing to copy objects below. Use different shutil.copy line to copy if `_exst_obj` is True